tests/fixtures/poetry-dev-dep.txt

```diff
@@ -0,0 +1,30 @@
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
this is a lock file from poetry <1.3 :white_check_mark: 

please **also** add a lock file from a newer poetry version.